Sources/MapboxDirections/RoadClasses.swift

```diff
@@ -43,6 +43,21 @@ public struct RoadClasses: OptionSet, CustomStringConvertible {
      */
     public static let tunnel = RoadClasses(rawValue: 1 << 5)
     
+    /**
+     The road segment is a HOV-2 road that requires a minimum of two vehicle occupants.
+    */
+    public static let hov2 = RoadClasses(rawValue: 1 << 6)
```

Comment:
The term “HOV” is a little more widely known than “HOT” but still somewhat obscure, since some countries and U.S. states use other terminology, like “carpool lane”. For consistency with `highOccupancyToll`, spell these cases out as `highOccupancyVehicle{2,3}` and link to [this Wikipedia article](https://en.wikipedia.org/wiki/High-occupancy_vehicle_lane) or [this OpenStreetMap Wiki article](https://wiki.openstreetmap.org/wiki/Key:hov). Similarly, we should clarify that the option pertains to HOV-only roads, not mere HOV lanes.